guide/hotel_reservation/tests/hotel_reservation_service_test.bal

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2018 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied. See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+import ballerina/test;
+
+// Common request Payload
+json requestPayload = {
+    ArrivalDate : "2007-11-06",
+    DepartureDate : "2007-11-06",
+    Location : "Changi"
+};
+
+// Client endpoint
+endpoint http:Client clientEP {
+    url:"http://localhost:9092/hotel"
+};
+
+// Function to test resource 'elizabeth'
+@test:Config
+function testResourceElizabeth () {
+    // Initialize the empty http requests and responses
```

Comment:
Here you only create a new request, not a response. This comment should be corrected in all the places it's been used this way.